setup.py

```diff
@@ -199,7 +199,7 @@ def run(self):
 ]
 
 extra_requirements = {
-    'test': ['pytest', 'pytest-cov', 'ipykernel', 'jupyter_client>=4.2'],
+    'test': ['pytest', 'pytest-cov', 'ipykernel', 'jupyter_client>=4.2', 'ipywidgets'],
```

Comment:
Does this need to be a specific version?

Reply:
should probably be at least >=7